checks/binary_artifact.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"fmt"
+
+	"github.com/h2non/filetype"
+	"github.com/h2non/filetype/types"
+
+	"github.com/ossf/scorecard/checker"
+)
+
+//nolint
+func init() {
+	registerCheck(CheckBinaryArtifacts, BinaryArtifacts)
+}
+
+const CheckBinaryArtifacts string = "Binary-Artifacts"
+
+// BinaryArtifacts  will check the repository if it contains binary artifacts.
+func BinaryArtifacts(c *checker.CheckRequest) checker.CheckResult {
+	return CheckFilesContent(CheckBinaryArtifacts, "*", false, c, checkBinaryFileContent)
+}
+
+func checkBinaryFileContent(path string, content []byte,
+	logf func(s string, f ...interface{})) (bool, error) {
+	binaryFileTypes := map[string]bool{
```

Comment:
Make this a const defined out the fn definition. Also, maybe specify one filetype per-line for better readability:

```
const binaryFileTypes := map[string]bool {
  "crx": true,
  "deb": true,
  ...
}
```

Reply:
golang does not allow const maps. https://www.reddit.com/r/golang/comments/993ujz/why_is_not_possible_to_create_a_constant_map/

Reply:
formatting the filetype per line. Thanks